Ruby/Ruby.sublime-syntax

```diff
@@ -289,9 +289,8 @@ contexts:
         2: invalid.illegal.character.ruby
 
   blocks:
-    - match: \b(do)\b\s*
-      captures:
-        1: keyword.control.start-block.ruby
+    - match: \bdo\b
+      scope: keyword.control.block.do.ruby
```

Comment:
Hm, what about using `keyword.control.context` from #1228 here?

Reply:
possible.

Reply:
The question is just whether we want to create some kind of relationship between `punctuation.section.block` and keywords which basically do the same - opening/closing blocks of code. If not, `context` is very fine as well here.